### ISSUE TITLE
Update check doesn't work correctly

### ISSUE THREAD

**Issue by ThomasBott** [EventGhost/EventGhost#74]
After installing the beta 1 of EG 0.5.0, I get this message:
It's German and it tells you that I'm not running the latest version, and that the latest version would have a "v" in front of it...
![versioncheck](https://cloud.githubusercontent.com/assets/1479181/17642998/b321937e-615c-11e6-822c-f325dc30c827.PNG)

**Comment by topic2k** [comment]
It seems like there is a issue with timezones and/or daylight saving times.

**Comment by blackwind** [comment]
The reason we didn't catch this is because of f1d834fc084773bf8ffb4532973d91b49e1a9c0d. Previously, we tagged the commit before the changelog commit, and the release time on GitHub matches the time of the tagged commit, so comparing times was sufficient, as the last commit would always come before the build. Yesterday, I committed f1d834fc084773bf8ffb4532973d91b49e1a9c0d and moved the tag to the changelog commit (because that'll be necessary if we need to put out a patch release for a specific version), and because the changelog commit will always come *after* the build, we end up with this scenario.

Bear with us for a day or two. I'll release beta2 when @topic2k sends along a fix.
